source/jobs/prerequisites/oracle.txt

```diff
@@ -28,13 +28,18 @@ About this Task
 
 - If you're migrating from an Oracle 12c instance, you must run commands as 
   the SYSDBA role.
-- In Oracle 12c the concept of a pluggable database was introduced. Some
-  of these commands can be run on PDB(plugable database) while commands 
+- Oracle 12c introduced the concept of a pluggable database. Some
+  of these commands can be run on a PDB (pluggable database), while commands 
```

Comment:
```suggestion
 necessary commands can be run on a PDB, while commands 
```

Reply:
Enabling archive logging is still "necessary," and is the reason you can't use Oracle DB XE for continuous jobs.